pages/cockpit/installation-guide/configuration/repositories/installation-guide-repositories-jdbc.adoc

```diff
@@ -0,0 +1,80 @@
+[[gravitee-cockpit-installation-repositories-jdbc]]
+= Configure a JDBC / R2DBC repository
+:page-sidebar: cockpit_sidebar
+:page-permalink: cockpit/1.x/cockpit_installguide_repositories_jdbc.html
+:page-folder: cockpit/installation-guide/repositories
+:page-description: Gravitee.io Cockpit - Repositories - JDBC
+:page-keywords: Gravitee.io, API Platform, API Management, Cockpit, documentation, manual, guide, reference, api, jdbc
+
+== Overview
+
+This repository plugin is for connecting to SQL databases.
+
+== Supported databases
+
+|===
+|Database | Version tested | JDBC Driver | R2DBC Driver
+
+|Postgresql
+|9.6
+|https://jdbc.postgresql.org/download.html[Download page]
+|https://repo1.maven.org/maven2/io/r2dbc/r2dbc-postgresql/0.8.7.RELEASE/r2dbc-postgresql-0.8.7.RELEASE.jar[Download page]
+
+|MySQL
+|5.7
```

Comment:
As we have repositories tests for mysql 5.7 and 8 do you think we could write something like ``5.7 and above`` or ``5.7 to 8``